### ISSUE TITLE
Cannot drop files from Explorer into WebView2 in a WinUI3 app

### ISSUE THREAD

**Issue by michaelcapps** [microsoft/microsoft-ui-xaml#7366]
### Describe the bug

When using a WebView2 in a WinUI3 application the drag and drop files from File Explorer into the WebView2 functionality behaves strangely. When displaying a page that is capable of receiving files (via copy when dropping, ex: https://www.remove.bg/ ) initially I am unable to drop files into the browser at all (a red circle with a link through it appears). However, if I click into the browser to manually choose a file through the explorer/file picker, this enables the usual drag and drop behavior, even after that explorer is closed. If I then minimize the application and maximize it again (or interact with a non-browser UI element, such as clicking a button to open a flyout), I am unable to drag and drop again.

### Steps to reproduce the bug

Make a new blank WinUI3 project and replace the default button with a WebView2, and set the source to a page that has a file drop control on it (such as remove.bg). Try to drag an image from File Explorer into the browser.

### Expected behavior

Dragging files from the file explorer into the browser copies the file to the app.

### NuGet package version

WinUI 3 - Windows App SDK 1.1.2

### Windows app type

- [ ] UWP
- [ ] Win32

### Device form factor

Desktop

### Windows version

Windows 10 (21H2): Build 19044

### Additional context

I reported this in the WebView2 repo and they directed me here (https://github.com/MicrosoftEdge/WebView2Feedback/issues/2546 ). This is not a new bug, it has been occurring since at least 0.8.

**Comment by michaelcapps** [comment]
Any update on this? Issue persists in WindowsAppSDK 1.2.220909.2-experimental2

**Comment by kbuckinx** [comment]
I see the same behaviour in WindowsAppsSDK version 1.2.221109.1

**Comment by michaelcapps** [comment]
Still seeing this in the latest stable 1.2 version and in the prerelease 1.3.230202101-experimental1 version.

**Comment by koenvd** [comment]
Would be nice if we could get this somehow on the roadmap to get fixed @bpulliam .

**Comment by codendone** [comment]
[internal task](http://task.ms/41952503)

**Comment by codendone** [comment]
@maxibyte Work is currently in progress on this, but there have been a series of issues and it is currently unclear how many more issues need to be solved to get this working. There also might still be fundamental blockers which could require larger feature work. In short, timeline isn't yet clear, other than the good news that this is currently being worked on.

**Comment by koenvd** [comment]
Wondering if there's already a better view on timeline when a fix would become available? Thanks!

**Comment by RicoSuter** [comment]
This is a huge blocker for us - we use Blazor + MAUI to build a desktop app and this is now preventing us to use drag and drop in the app. It's not just a tiny feature but everything drag and drop related cannot be done... 

Ref: https://github.com/MicrosoftEdge/WebView2Feedback/issues/2805

**Comment by michaelcapps** [comment]
I just tested this in 1.4.230913002 and it's still broken, so I don't know why this was marked as fixed.

**Comment by michaelcapps** [comment]
> @michaelcapps this is fixed in main branch so the fix will be available in the next release from main (1.5 experimental)

Ah I completely missed it being added to the 1.5 milestone, sorry about that. Thank you!

**Comment by koenvd** [comment]
Based on release notes https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/stable-channel#version-15-1 this got moved to 1.6 @codendone ? But is there any timeline around that already available?

**Comment by codendone** [comment]
@koenvd It looks like we missed some detailed in the release notes:
* Dragging **into** a WebView2 should now be functional in 1.5. A fix for this was checked into 1.5 shortly after 1.4 was released.
* Dragging **out** of a WebView2 (or even within a WebView2, if that drag uses drag-n-drop) is still not fixed and got moved to 1.6. Fixing dragging out requires work both in core WebView2 and the XAML element. That work is still in progress.

**Comment by R-Vulcan** [comment]
I find that with MAUI Blazor Hybrid its still not possible to drag a file onto a html file input element.

I recently ported a Blazor WASM app to Blazor Hybrid and the file input drag drop stopped working.
I use MudBlazor but I've been able to reproduce this with the native element and InputFile in a separate project, just adding the input to the default Blazor Hybrid template.

.NET8 + Microsoft.AspNetCore.Components.WebView 8.0.71 -> Microsoft.WindowsAppSDK 1.5.240607001
